lib/widget/input/form_textfield.dart

```diff
@@ -317,6 +316,14 @@ class TextInputState extends FormFieldWidgetState<BaseTextInput>
         }
       }
     });
+
+    // Checking for readOnly from parent widget and assign the value to TextInput and PasswordInput if it's readOnly property is null
+    final formController =
```

Comment:
Dont' waste unnecessary cycles. There's no need to traverse the tree unless widget._controller.readOnly is not set.

Reply:
Updated it